policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml

```diff
@@ -0,0 +1,15 @@
+description: Montana allows adjustments from US AGI of the lesser of US-taxable social security and an alternative amount with the complex calculation including this fraction which is applied to total social security benefits.
```

Comment:
```suggestion
description: Montana multiplies the smaller of the income cap fraction and the fraction of federal social security benefits.
```

Reply:
federal social security benefits is line 1 and 2 right?

Reply:
> federal social security benefits is line 1 and 2 right?
Yes, line 1 is the net amount of ss benefits and line 2 is a half of that.

Reply:
> federal social security benefits is line 1 and 2 right?

Yes, line 1 is the net amount of ss benefits and line 2 is a half of that.